problems/enumerated/problem_10384/problem_10384.py

```diff
@@ -0,0 +1,306 @@
+from manim import *
+from lib.aramanim import *
+from objects import Scissors
+from .text import *
+
+class Problem10384(Scene):
+	def construct(self):
+		self.wait()
+		self.set_up()
+
+		self.play(Write(self.task_number))
+
+		self.wait()
```

Comment:
Not use empty lines (enters)
```suggestion
		self.play(Write(self.task_number))
		self.wait()
```